instructor/retry.py

```diff
@@ -64,7 +64,12 @@ def retry_sync(
     strict: Optional[bool] = None,
     mode: Mode = Mode.TOOLS,
 ) -> T_Model:
-    total_usage = CompletionUsage(completion_tokens=0, prompt_tokens=0, total_tokens=0)
+    total_usage = CompletionUsage(
```

Comment:
We can definitely move it somewhere else, but we shouldn't be moving it into the usage.

Reply:
Cool. What would you suggest? I played around with this today and it feels like a simplified version of DSPy since both approaches give us a better input for future calls. So it's definitely useful.